ports/stm32/mboot/mboot_pack_dfu.py

```diff
@@ -228,11 +230,22 @@ def verify_pack_dfu(keys, filename):
                 chunk, 0, MBOOT_PACK_HYDRO_CONTEXT, keys.secretbox
             )
             assert chunk is not None
+            binary_data += chunk
             if header[1] == MBOOT_PACK_CHUNK_FW_GZIP:
                 chunk = zlib.decompress(chunk, wbits=-15)
             full_sig.update(chunk)
     full_sig_pass = full_sig.final_verify(actual_sig, keys.sign_pk)
     assert full_sig_pass
+    return binary_data
```

Comment:
This should return a list of tuples of addr, data, to feed into `dfu.build()`.  But I think it's ok for now to just support one element, ie just one contiguous chunk with one address.

That address would be the first address encountered in the for loop above, eg `base_addr = addr`.  And then to assert that there's only one element, subsequent iterations of the for loop would assert `addr == base_addr + len(binary_data)`.  Then at the end return `[{"address": base_addr, "data": binary_data}]`.

Reply:
Done